homework/02LessonTwo.js

```diff
@@ -74,22 +74,28 @@ function addThree(number1, number2, number3){
 // What is a function? How do you define a function in Javascript?
 //
 // Your Answer Goes Here:
-//
+//a function is a method that is called
+  //it allows you to call reusable snippets of code
+    //eliminates repetition and drys up code
+      //function is defined by writing 'function funcName(){}'
 //
 // What is a return value?
-//
 // Your Answer Goes Here:
+//The value returned by a function
 //
 //
 // How do you define a named function?
 //
 // Your Answer Goes Here:
-//
+//A function that has a name like funcName(){} as opposed to a nameless function like a callback function(){}
 //
```

Comment:
To define a named function:

**function** funcName(parameters){}

Here is some interesting reading i found about the difference between a named function and an anonymous function: http://stackoverflow.com/questions/10081593/are-named-functions-or-anonymous-functions-preferred-in-javascript